docs/node/run-your-node/archive-node.md

```diff
@@ -68,9 +58,9 @@ Output should report `archive` consensus mode status:
 ```json
 {
   // other fields omitted ...
-  "consensus": {
-    "mode": "archive",
-    // ...
+  "mode": "archive",
```

Comment:
TODO: this is actually missing currently in control status output in 23.x

Add a note that this is fixed in release XXX

Reply:
done